web/chromecom.js

```diff
@@ -171,6 +168,7 @@
 
   var chromeFileAccessOverlayPromise;
   function requestAccessToLocalFile(fileUrl) {
+    var overlayManager = PDFViewerApplication.overlayManager;
```

Comment:
Nit: I'm wondering if it might not be a bit cleaner to just have this definition at the top of the file instead (like it currently is).